recipes/bioconductor-transite/meta.yaml

```diff
@@ -12,18 +12,18 @@ source:
     - 'https://depot.galaxyproject.org/software/bioconductor-{{ name }}/bioconductor-{{ name }}_{{ version }}_src_all.tar.gz'
   md5: 6e8830d4874924cdb4245eaade39de90
 build:
-  number: 1
+  number: 2
   rpaths:
     - lib/R/lib/
     - lib/
 # Suggests: knitr (>= 1.20), rmarkdown (>= 1.10), roxygen2 (>= 6.1.0), testthat (>= 2.1.0)
 # SystemRequirements: C++11
 requirements:
   host:
-    - 'bioconductor-biocgenerics >=0.36.0,<0.37.0'
-    - 'bioconductor-biostrings >=2.58.0,<2.59.0'
-    - 'bioconductor-genomicranges >=1.42.0,<1.43.0'
-    - r-base
+    - 'bioconductor-biocgenerics >=0.26.0'
```

Comment:
Bioconductor packages are not compatible between releases, we can't relax these constraints at all.

Reply:
Ah, got it, thanks 👍